spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-specialist-link-tracker.spec.js

```diff
@@ -740,7 +740,7 @@ describe('A specialist link tracker', function () {
     })
 
     afterEach(function () {
-      GOVUK.cookie('cookies_policy', null)
+      GOVUK.setCookie('cookies_policy', null)
```

Comment:
I thought that `.cookie` and `.setCookie` were equivalent, is that not the case? Was this causing an error?